views/partials/header.handlebars

```diff
@@ -1,5 +1,5 @@
 <header class="nav_bar">
-    <img id = "logo" src="assets/popsicle.svg" />
+    <img id = "logo" src="/assets/popsicle.svg" />
```

Comment:
What’s up with this path?

Reply:
it didn't work with the old path, idk I was getting a 404 error with the older one

Reply:
It should be fixed by the new branch Thuyv-Vy and I are working on.

Reply:
bro no one has ever spelled my name like that before what